### ISSUE TITLE
Null attribute values appears as false

### ISSUE THREAD

**Issue by Owemyr** [origo-map/origo#1196]
**Describe the bug**
Null values now appears as "false" in feature overlay and infowindow when using attribute configs as below.

` {
          "html": "Namn på url",
          "url": "url_attribute"
        },`

**Expected behavior**
Not display attribute when null.

**Screenshots**
![probs](https://user-images.githubusercontent.com/47249045/110590324-ed76a880-8177-11eb-9959-79a836a1ac8e.PNG)

**Additional context**
Not sure but I think PR #1188 migth have introduced the issue.

**Comment by jokd** [comment]
Are you using geopackage as data source? Is this relevant: https://github.com/origo-map/origo/issues/1103 ?

**Comment by johnnyblasta** [comment]
Have you updated so that PR #1195 is included?
#1103 is not relevant since that is only in effect for default template and not when attributes is specified.

**Comment by Owemyr** [comment]
> Are you using geopackage as data source? Is this relevant: #1103 ?
Using geoserver.

**Comment by Owemyr** [comment]
@johnnyblasta yes using latest master

**Comment by johnnyblasta** [comment]
> @johnnyblasta yes using latest master

There is a PR #1199 which will hopeful fix this, which you can test now.

**Comment by Owemyr** [comment]
Yes null values working fine in my usecase with PR #1199, thanks.
